src/main/java/de/retest/recheck/ui/diff/Warning.java

```diff
@@ -0,0 +1,60 @@
+package de.retest.recheck.ui.diff;
+
+import java.io.Serializable;
+
+import javax.xml.bind.annotation.XmlAttribute;
+
+import lombok.Data;
+
+/**
+ * The class Warning stores structurally the information about a test breakage. This information can be given in the
+ * review-GUI and recheck.cli, when someone applies a change.
+ */
+@Data
+public class Warning implements Serializable {
+
+	private static final long serialVersionUID = 1L;
+
+	@XmlAttribute
+	private final String elementIdentifier;
+
+	@XmlAttribute
+	private final String byMethodName;
+
+	@XmlAttribute
+	private final Object oldValue;
+
+	@XmlAttribute
+	private final Object newValue;
+
+	@XmlAttribute
+	private final String test;
+
+	@XmlAttribute
+	private final String callLocation;
+
+	@XmlAttribute
+	private final String retestId;
+
+	@SuppressWarnings( "unused" )
+	private Warning() {
+		elementIdentifier = null;
+		byMethodName = null;
+		oldValue = null;
+		newValue = null;
+		test = null;
+		callLocation = null;
+		retestId = null;
+	}
+
+	public Warning( final String elementIdentifier, final String byMethodName, final Object oldValue,
```

Comment:
Should be generated by `@Data` (see [here](https://projectlombok.org/features/Data)).